### ISSUE TITLE
Check for Java installation

### ISSUE THREAD

**Issue by lubocode** [gekigek99/minecraft-server-hibernation#41]
Since you need to have java installed to run a Minecraft server it might be nice to inform the user if the Java installation is not found or the version is insufficient.
A simple `java -version` command in cmd/bash works on Windows and Linux and prints the current Java version along some other info.
Since this is probably language specific, it might be hard to parse the string that comes back.
A first step would then be to log the output so that the user can at least interpret it themselves.

**Comment by gekigek99** [comment]
added in bc373ba84a02f4ab1b85202dc47fde22ce54ea4a

**Comment by lubocode** [comment]
@gekigek99 did you test this on Linux? I don't think the console returns an error, if something is not installed, but rather just displays text which indicates that the package was not found.

I suppose on Windows it works, since you have that as your main system?

**Comment by gekigek99** [comment]
Yes on Windows it works... On Linux I could not test it (i don't have a working Linux server right now)

**Comment by lubocode** [comment]
Ok, I will test it this week, though I expect it NOT to work.
If it does not work, I think I should fix it, as I can test the code right away.

Notes for myself:
https://stackoverflow.com/questions/48253268/print-the-stdout-from-exec-command-in-real-time-in-go
https://stackoverflow.com/questions/30993717/how-can-i-redirect-the-stdout-and-stderr-of-a-command-to-both-the-console-and-a?rq=1

**Comment by lubocode** [comment]
```
 _ __ ___  ___| |__
| '_ ` _ \/ __| '_ \
| | | | | \__ \ | | |
|_| |_| |_|___/_| |_|
Copyright (C) 2019-2020 gekigek99
v2.0.0 (Go)
visit my github page: https://github.com/gekigek99
remember to give a star to this repository!
2020/11/02 15:49:36 loadConfig: config error: java not installed!
```

Seems to work. I will close this and we can reopen it, if anyone notices a bug with this in the future.
(Checking for server file works as well)
